mappings/net/minecraft/block/Material.mapping

```diff
@@ -4,45 +4,45 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 DECORATION Lnet/minecraft/class_3614;
```

Comment:
No, this is used by more than just decroation blocks?

Reply:
I'm open to suggestions, but PART isn't very descriptive

Reply:
Used for skulls, torches, potted plants, ladders, redstone components, and rails.
Perhaps `SUBBLOCK`?

Reply:
like fluid loggable block? redstone components and rails are circuits

Reply:
None of the blocks in this material are fluid loggable (except maybe ladders).
If `SUBBLOCK` is already in use for fluid logging then that won't work for a material name in this case. `PART` seems like a multipart or part of something else, which these are not. Initially chose `DECORATION` as that's what's on the wiki (per the original issue), and `PART` is just too vague.